### ISSUE TITLE
Unoptimised query Admin::EventsController#events

### ISSUE THREAD

**Issue by denys281** [pivorakmeetup/pivorak-web-app#171]
In Admin::EventsHelper

```
  def event_limitation_data(event)
      total    = event.limit_total
      verified = event.visit_requests.approved.count
      pending  = event.visit_requests.pending.count
      .........
   end
```

![screen shot 2017-02-03 at 3 01 08 pm](https://cloud.githubusercontent.com/assets/754467/22592283/9fcc6382-ea21-11e6-8a3e-55bfd63905b3.png)

Probably in need to be refactored.

@sveredyuk

**Comment by sveredyuk** [comment]
Ofc it's should

**Comment by kushniryb** [comment]
Fixed in #185
